hedera-node/src/test/java/com/hedera/services/state/migration/MigrationRecordsManagerTest.java

```diff
@@ -580,6 +580,37 @@ void traceabilityMigrationDoesNotAddStateChangesWhenSmartContractDoesNotHaveAnyS
                 sidecarRecords.build());
     }
 
+    @Test
+    void traceabilityMigrationDoesNotCreateMigrationRecordsForContractWithoutBytecodeInState() {
```

Comment:
Q: doesn't this need a test case for the null runtime_bytecode check added?

Reply:
Correct, it was just that the `null` return of [`entityAccess.fethCodeIfPresent()` was implicit by the mock.](https://github.com/hashgraph/hedera-services/blob/7140ff130efad054ac35d0f130f5364ed052d645/hedera-node/src/main/java/com/hedera/services/state/migration/MigrationRecordsManager.java#L360-L363)

Added    

```
given(entityAccess.fetchCodeIfPresent(contractEntityNum.toGrpcAccountId()))
                .willReturn(null);
```

for better clarity.